README.md

```diff
@@ -28,5 +28,14 @@ Just a simple game about collecting stars and not dying :)
 13. Win death feature
 14. Added Readme
 
+## Comments
+- [index.js:77](https://github.com/erhuz/Canvas/blob/master/src/index.js#L77) - Remember to remove unused code
+- [index.js:122](https://github.com/erhuz/Canvas/blob/master/src/index.js#L122) - You could probably move the room blocks into separate files / functions. Or build a JSON file for each room and build the level with a loop. `*.create(json)` etc
+- [index.js:132](https://github.com/erhuz/Canvas/blob/master/src/index.js#L132) - Maybe remove some whitespace?
+- [index.js:234](https://github.com/erhuz/Canvas/blob/master/src/index.js#L234) - Unused 'header'
```

Comment:
Header is not unused but should probably be refactored to `END CREATE WORLD` instead of `/CREATE WORLD`